README.md

```diff
@@ -23,9 +23,17 @@ Usage
 Commands
   validate 
    Options
-     -c --context  Your AsyncAPI specification
+     -c --context  context-name saved in the store
      -w --watch Enable watchMode (not implemented yet)
+  
+  context
+    current show the current set context
+    list show the list of all stored contexts
+    remove <context-name> remove a context from the store
+    use <context-name> set any context from store as current
+    add <context-name> <filepath> add/update new context
 
 Examples
-  $ asyncapi validate --context=specification.yml
+  $ asyncapi context add dummy ./specification.yml
```

Comment:
let us always promote naming with `asyncapi`
```suggestion
  $ asyncapi context add dummy ./asyncapi.yml
```

also please rename example spec in this repo